solutions/arrays/noble-integer.js

```diff
@@ -0,0 +1,34 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+

```

Comment:
@DevratAgarwal  PR title should be **Array - Noble integer** as per the PR title naming conventions.